flow/Makefile

```diff
@@ -1,88 +1,7 @@
-# ==============================================================================
-# Uncomment or add the design to run
-# ==============================================================================
+# settings.mk is not under source control. Put variables into this
+# file to avoid having to adding the to the make command line.
+-include settings.mk
 
-# DESIGN_CONFIG=./designs/nangate45/aes/config.mk
-# DESIGN_CONFIG=./designs/nangate45/ariane136/config.mk
-# DESIGN_CONFIG=./designs/nangate45/black_parrot/config.mk
```

Comment:
We've already discussed not removing these previously.  Please restore them.

Reply:
Will do. Do you mind if I force push, so I can simply delete this commit? Otherwise there will be delete + revert commit.

Reply:
Ok as I haven't looked further.